src/views/watershed/WatershedIntroduction.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <div class="aside-content">
-    <h1>Watershed Intoduction</h1>
-    <div id="lipsum">
-      <Accordion :open="true">
+    <div class="title"><p><h1>Outside in</h1></p></div>
+    <img src="../../../public/images/waterbody.png"><br><div class="caption">Does the territory inhabit you?</div><br>
```

Comment:
FYI:  the difference between src/assets/images and public/images/ is that files inside of  src/assets/ will only end up in the build if they are referenced, so that's why before the src attributes backed up to reference the assets/ directory. alternatively, everything inside of the public/ dir will end up in the build so you can reference those files like they get served from the webserver (a la http://localhost:8080/images/waterbody.png or on whatever domain) 

the idea with the assets/ directory is to only build and include the files that are actually referenced in code. but i just dumped everything in the public/ dir because the LightBox, map icons image, and geojson src references don't get picked up in the build process.